landing-pages/site/assets/scss/_typography.scss

```diff
@@ -107,3 +107,12 @@ $bodytext-styles: (
 .text-break-all {
     word-break: break-all
 }
+
+/* Fix for Announcements Page line-height */
+.td-main.container {
```

Comment:
This CSS selector looks quite broad and seems to affect multiple areas such as the Blog and Home pages.
Additionally, since it is defined with `!important`, it becomes difficult to override the style in more specific contexts.